subnets.tf

```diff
@@ -1,18 +1,18 @@
-# Copyright 2017, 2021 Oracle Corporation and/or affiliates.  All rights reserved.
+# Copyright 2017, 2022 Oracle Corporation and/or affiliates.  All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
 
 resource "oci_core_subnet" "operator" {
   cidr_block                 = local.operator_subnet
   compartment_id             = var.compartment_id
   display_name               = var.label_prefix == "none" ? "operator" : "${var.label_prefix}-operator"
-  dns_label                  = "operator"
+  dns_label                  = var.assign_dns ? "operator" : null

```

Comment:
In the OKE module, we are still keeping the label prefix e.g. https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/modules/network/subnets.tf#L8 

Shouldn't we keep this the same?